KnockKnock-iOS/Scenes/Cells/FeedCell.swift

```diff
@@ -27,7 +28,17 @@ class FeedCell: BaseCollectionViewCell {
 
   let severalSymbolImageView = UIImageView().then {
     $0.translatesAutoresizingMaskIntoConstraints = false
-    $0.backgroundColor = .white
+    $0.image = KKDS.Image.ic_more_img_20_wh
+    $0.isHidden = true
+  }
+
+  // MARK: - Bind
+
+  func bind(feed: Feed) {
+    if feed.images.count > 1 {
+      self.severalSymbolImageView.isHidden = false
+    }
```

Comment:
```suggestion
    if feed.images.count > 1 {
      self.severalSymbolImageView.isHidden = false
    } else {
      self.severalSymbolImageView.isHidden = true
    }
```
이것도 사실 재사용시에 문제가 될 수 있어요 
이미지 갯수가 1 이상이라면 다음과 같이 `self.severalSymbolImageView.isHidden = false` 처리를 해주고 있는데 저 조건에 해당되지 않는 경우의 cell 에서도 isHidden이 false 나타나는 케이스가 발생가능합니다.

Reply:
+ 추가로 더 간단한게 표현한다면 
```suggestion
    self.severalSymbolImageView.isHidden = feed.images.count <= 1 
```